tests/functional/ionIonBeam/ion_ion_beam1d.py

```diff
@@ -125,16 +130,19 @@ def growth_b_right_hand(run_path):
         x = xyz_finest[0][:-1]
 
         by = by_interpolator(x)
-        bz = by_interpolator(x)
+        bz = bz_interpolator(x)
 
         # get the mode 1, as it is the most unstable in a box of length 33
         mode1 = np.absolute(np.fft.fft(by-1j*bz)[1])
         first_mode = np.append(first_mode, mode1)
+    
```

Comment:
add a comment :
the find_peaks method is used to identify the max value of the time serie of the Fourier mode m=1. this max being an overshoot, the width of this overshoot is delta_t=10, which is an empirical value